src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -51,28 +52,33 @@ export default function DownloadDetails(props: DownloadDetailsProps) {
   }, [accessToken])
 
   const timeEstimateInSeconds =
-    isLoading || downloadSpeed === 0 ? 0 : numBytes / downloadSpeed
+    isLoading || downloadSpeed === 0 || !numBytes ? 0 : numBytes / downloadSpeed
+  const isTimeEstimateLoading = timeEstimateInSeconds === 0
```

Comment:
Tried to maintain the same logic that was already in this component ( they seem to use timeEstimateInSeconds=0 to indicate that it should be inactive)

Reply:
Can be hard to follow, but it at least keeps this change simple